ldrshared/clients/messenger/pubsub.py

```diff
@@ -90,8 +95,10 @@ def store(self, message: Message):
             self._housekeeping()
         else:
             # Malformed time stamp, ignore but log it
-            logging.warning(f'Malformed message passed to store: {message_as_dict}, '
-                f'"time_stamp" key was {message.attributes.get(TIME_STAMP_KEY)}')
+            logging.warning(
```

Comment:
Now this doesn't matter so much for warning messages since any sane person would have the log level set to a most `warning`, but I'm against using string interpolation in logging messages like this. 

The reason I'm against using f-strings is because the f-string is built even if we're not logging at that level. Which means that doing potentially expensive operations like converting that whole (multi-level) dict to a string happens regardless of whether the message ends up being visible anywhere.

You can use the `%`-encoding functionality built in to the logging framework to avoid having to do the expensive stuff like casting objects to strings if the log message gets thrown away. [See python docs for examples](https://docs.python.org/3/howto/logging.html#logging-variable-data). Essentially this approach takes something that's eagly-evaluated and turns it into lazy-evaluation.

So something like the following is how I would prefer to see most logging statements written:

```python
logging.warning("Malformed message passed to store: %s, "time_stamp" key was %s", message_as_dict, message.attributes.get(TIME_STAMP_KEY))
```

But again, since this is at the warning level it's not as bad as the same thing happening in a debug/info level statement.